src/app/services/article-list.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
ta część powoduje, że nie musimy dodatkowo umieszczać deklaracji w `providers` w komponencie czy module. Samo to mówi, że ma być dostępne globalnie więc instancja jest tworzona dla wszystkich :)